core/src/test/scala/org/apache/spark/sql/delta/optimize/OptimizeMetricsSuite.scala

```diff
@@ -295,6 +307,8 @@ trait OptimizeMetricsSuiteBase extends QueryTest
 
         val expMetrics = JsonUtils.fromJson[Map[String, String]](expMetricsJson)
         assert(actualMetrics === expMetrics)
+
+        assertOperationName(history)
```

Comment:
we don't need to verify the history in every optimize test (increased test run time). I think the two tests which specifically test the history commit info, should also check the operation name. Let me know.

Reply:
I removed it from "deletion vector metrics" test. Is that what you meant?